widgets/src/components/common/DropDownButton.js

```diff
@@ -46,13 +46,7 @@ class DropDownButton extends React.Component {
 
     return (
       <>
-        <ButtonGroup
-          color="primary"
-          ref={this.anchorRef}
-          disableElevation
-          disabled={disabled}
-          variant="outlined"
-        >
+        <ButtonGroup color="primary" ref={this.anchorRef} disabled={disabled} variant="outlined">
```

Comment:
on Figma designs, button doesn't have elevation, why have u removed `disableElevation` here?

Reply:
`ButtonGroup` does not support `disableElevation` (https://material-ui.com/api/button-group/), that leads to an error in console as it's passed down to DOM element.